3.0/README.md

```diff
@@ -39,28 +39,109 @@ Implementations MUST treat unknown keys as if they weren't present. However, imp
 
 ## 3.1 `attribution`
 
-REQUIRED|OPTIONAL (description of dependents/dependencies)
+OPTIONAL. Default: null.
 
-**Description:**
+Contains an attribution to be displayed when the map is shown to a user. Implementations MAY decide to treat this as HTML or literal text. For security reasons, make absolutely sure that this field can't be abused as a vector for XSS or beacon tracking.
 
-**Example:**
+```JSON
+{
+  "attribution": "<a href='http://openstreetmap.org'>OSM contributors</a>"
+}
+```
 
 ## 3.2 `bounds`
 ## 3.3 `center`
 ## 3.4 `data`
 ## 3.5 `description`
+
+OPTIONAL. Default: null.
+
+A text description of the tileset. The description can contain any legal character. Implementations SHOULD NOT interpret the description as HTML.
+
+```JSON
+{
+  "description": "We are volcanoes. When we women offer our experience as our truth, as human truth, all the maps change. There are new mountains."
+}
+```
+
 ## 3.6 `grids`
 ## 3.7 `legend`
+
 ## 3.8 `maxzoom`
+
+OPTIONAL. Default: 30. >= 0, <= 30.
+
+An integer specifying the maximum zoom level. MUST be >= minzoom.
+
+```JSON
+{
+  "maxzoom": "11"
+}
+```
+
 ## 3.9 `minzoom`
+
+OPTIONAL. Default: 0. >= 0, <= 30.
+
+An integer specifying the minimum zoom level.
```

Comment:
Since `maxzoom` has "MUST be >= minzoom" do we want to include the same for `minzoom`? "MUST be <= maxzoom"